doc/09_Releaseing.rst

```diff
@@ -93,12 +93,12 @@ Commit and push your changes.
 
    git add setup.py README.rst
    git commit -m "updated setup.py and readme (#2)"
-   git checkout master
-   git merge --no-ff 2-implement-ui
-   git push origin master
+   git checkout main
+   git branch --merged | grep -i -v -E "master|main" | xargs git branch -d
```

Comment:
What's this command doing? Will these pipes only work on Linux?

Reply:
`grep` might not work on Github and this line might be advance. We can be more specific to the particular branch of this tutorial. It might be good to add instead links for git advance commands.